rust/main/agents/relayer/src/msg/op_batch.rs

```diff
@@ -44,10 +45,14 @@ impl OperationBatch {
         };
 
         if !excluded_ops.is_empty() {
-            warn!(excluded_ops=?excluded_ops, "Either operations reverted in the batch or the txid wasn't included. Falling back to serial submission.");
-            OperationBatch::new(excluded_ops, self.domain)
-                .submit_serially(prepare_queue, confirm_queue, metrics)
-                .await;
+            warn!(excluded_ops=?excluded_ops, "Either operations reverted in the batch or the txid wasn't included. Sending them back to prepare queue.");
+            let reprepare = excluded_ops.into_iter().map(|mut op| {
```

Comment:
todo regular for since they'd block on the mutex either way

Reply:
Done